src/main/java/no/nav/fo/veilarbregistrering/registrering/bruker/InaktivBrukerService.java

```diff
@@ -31,7 +34,11 @@ public void reaktiverBruker(Bruker bruker) {
         }
 
         brukerRegistreringRepository.lagreReaktiveringForBruker(bruker.getAktorId());
-        oppfolgingGateway.reaktiverBruker(bruker.getGjeldendeFoedselsnummer());
+        AktiverBrukerResultat aktiverBrukerResultat = oppfolgingGateway.reaktiverBruker(bruker.getGjeldendeFoedselsnummer());
+
+        if (aktiverBrukerResultat.erFeil()) {
+            throw new WebApplicationException(JsonUtils.toJson(new AktiveringFeilResponse(aktiverBrukerResultat.feil().toString())));
```

Comment:
Hvis dette funker, så lurer jeg på om vi kunne ha introdusert en ny Exception internt i stedet som vi oversatte til, catchet for å oppdatere riktig status og oversatte tilbake på vei ut igjen (controller). 

Jeg lurer på om det kan gi en litt enklere løsning. Synes kanskje exception funker bedre enn returobjekter ved unntak. Men er ikke helt sikker med meg selv

Reply:
Det var tanken på sikt. Dette var en enklere løsning for å få noe ferdig.

Reply:
Vi snakket også om å heller returnere en vanlig response for manglende opphold og utvandret fordi det er "forventet" oppførsel.